types/node/index.d.ts

```diff
@@ -5969,7 +5968,7 @@ declare module "stream" {
             unshift(chunk: any): void;
             wrap(oldStream: NodeJS.ReadableStream): this;
             push(chunk: any, encoding?: string): boolean;
-            _destroy(err: Error, callback: Function): void;
+            _destroy(error: Error, callback: (error?: Error) => void): void;
```

Comment:
I tihnk `_destroy` may get `Error | null` as input; `null` in case `destroy()` is called.

Reply:
I had that initially, but it causes breaks in about 7 other packages. I might look into that in a later PR.